message-index/messages/GHC-28007/example1/after/Main.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE RecursiveDo #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
+
+import Prelude
```

Comment:
Do we need a manual Prelude import?

Reply:
no, the example case here is exactly what the test suite had

Reply:
I think in some cases we can differ from the testcases where they have redundant things (e.g. a `NoDatatypeContexts` pragma despite `DatatypeContexts` being a very not good extension)